docs/finalreport.Rmd

```diff
@@ -4,7 +4,7 @@ author: "Matthew Connell, Firas Moosvi"
 date: "05/03/2020"
 always_allow_html: true
 output: 
-  html_document:
+  bookdown::html_document2:
```

Comment:
@STAT547-UBC-2019-20/students : In order to get cross-referencing and figure referencing to work in Rmd files, you need to replace `html_document` in the YAML header with `bookdown::html_document2`.

Apologies, I missed that step in the class notes!